yarn.lock

```diff
@@ -87,6 +87,23 @@
     "@aws-amplify/api-graphql" "^1.2.5"
     "@aws-amplify/api-rest" "^1.2.5"
 
+"@aws-amplify/appsync-modelgen-plugin@^1.20.12-dev.136":
```

Comment:
The same for version change here. It should be the latest version

Reply:
done!!